### PR TITLE
Modmap: Where to put it; cancelling built-in mods (#655)

### PR DESCRIPTION
**Pull request by Spike-from-NH**

### CODE REVIEW

File: doc/Custom-layouts.md [hunk 1]
```diff
@@ -95,23 +95,30 @@ If the string defining a tap or a swipe is anything other than one of the built-
...
-### Example
+### Examples
 Turkish keyboards use the Latin alphabet, but when "i" is shifted, it should produce "İ". This is achieved with the following modmap: 
 
-
     <modmap>
         <shift a="i" b="İ" />
     </modmap>
+    
+On built-in layouts for European locales, `Fn`+`e` produces the € character. This can be inhibited with the following modmap, which maps the modified `e` key to itself:
```
Comment (Julow) [inline doc/Custom-layouts.md:115]: That's the case on every locales. On every layouts that have `Fn` and `e`, `Fn+e = €`.

**Review by Julow — COMMENTED** [review]
Awesome, thanks! As this is documentation, I've left one nick-pick. Otherwise, that's good to go.

File: doc/Custom-layouts.md (hunk 1 above)
Comment (Spike-from-NH) [reply doc/Custom-layouts.md]: I've deleted the qualification.

**Comment by Spike-from-NH** [comment]
Separately, is this section tight enough?
- I say that "Typically" a layout has only one modmap; should this be mandatory?
- I say the modmap is inside `<keyboard>` but outside any `<row>`. Should it be before any row?
Also: In **Swipes**, the boldfacing and shading of certain of the compass points suggests there is a distinction when there isn't. At most, the center cell should be highlighted. I don't know Markdown well enough to fix this.

**Comment by Julow** [comment]
> I say that "Typically" a layout has only one modmap; should this be mandatory?

Currently, the second modmap would erase the first one. This should be made into an error though and should be described as such.

> I say the modmap is inside <keyboard> but outside any <row>. Should it be before any row?

No constraint on that.

> In Swipes, the boldfacing and shading of certain of the compass points suggests there is a distinction when there isn't.

Table have a mandatory header row and this is a limitation of Github Markdown. Perhaps this can be fixed by writing a raw HTML table ?

**Review by Julow — APPROVED** [review]
Thanks! The HTML tables look much better.

**Comment by Spike-from-NH** [comment]
> Thanks! The HTML tables look much better.

But all tables seem to have a default style of shading every other row, which is inappropriate here—It implies there's something special about `w` and `e`.

**Comment by Julow** [comment]
That's fine !

**Pull request merged**